.gitignore

```diff
@@ -8,3 +8,4 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
```

Comment:
Could you remove this change?

Reply:
I can remove it.

But, I didn't write. 
When I use rustfmt, is it always written automaticaly?

Reply:
I can remove it.

But, I didn't write. 
When I use rustfmt, is it always written automaticaly?

Reply:
Sorry, I may be understood.

You tell me to remove neither
```
# These are backup files generated by rustfmt
**/*.rs.bk
```
nor new line.
Is this correct?

Reply:
I mean revert the change about this file.
(remove newline)